src/commands/database-settings-get.ts

```diff
@@ -0,0 +1,60 @@
+import { Response } from "request";
+import * as _ from "lodash";
+import * as request from "request";
+
+import * as responseToError from "../responseToError";
+import * as Command from "../command";
+import * as logger from "../logger";
+import * as FirebaseError from "../error";
+import * as requirePermissions from "../requirePermissions";
+import * as utils from "../utils";
+import * as api from "../api";
+import * as requireInstance from "../requireInstance";
+import { DATABASE_SETTINGS, DatabaseSetting } from "../database/settings";
+
+export default new Command("database:settings:get <path>")
+  .description(
+    "read the realtime database setting. The available settings are:\n" +
+      Array.from(DATABASE_SETTINGS.values())
+        .map((setting: DatabaseSetting) => `${setting.path}${setting.description}`)
+        .join("")
```

Comment:
This description isn't going to work. If you do `firebase help`, you can see that the descriptions for every other command are one line, but this one is now a huge paragraph. Worse, it's duplicated because of `get` and `set` both outputting the same thing.

I think there's a way with commander to override the help method of a single command to manipulate the help text. If you could have the available database settings to change show up when the user does `firebase help database:settings:get` (or `firebase database:settings:get --help`), that would be a better experience.

This applies for `get` and `set`.

Reply:
Bingo. Found out how to do it.
I added a `help` method to `command.js`, so that we can always inject additional help message that do not show up as description.